lib/editor/containers/ActiveTimetableEditor.js

```diff
@@ -47,8 +47,7 @@ const mapStateToProps = (state: AppState, ownProps: Props) => {
   const columns = getTimetableColumns(state)
   const tripValidationErrors = getTripValidationErrors(state)
   const {subEntity: activePattern} = active
-  const activeSchedule = getTableById(tables, 'calendar')
-    .find(c => c.service_id === activeScheduleId)
+  const activeSchedule = getTableById(tables, 'calendar').find(c => c.service_id === activeScheduleId) || getTableById(tables, 'scheduleexception').find(ex => ex.custom_schedule && ex.custom_schedule[0] === activeScheduleId)
```

Comment:
Will this result in a crash on bad data?

Reply:
It should be fine with the inclusion of the check for `ex.custom_schedule`